processing/src/main/java/org/apache/druid/query/rowsandcols/util/FindResult.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.druid.query.rowsandcols.util;
+
+public class FindResult
```

Comment:
"dealing in `int`" would actually mean `dealing in int[]` because I am often returning a start/end range.  Once we are dealing in `int[]`, we have the overhead of a reference to an object anyway, so this seemed okay.

If we really wanted to fix this, we'd likely need to make the finder thingie itself stateful and have getters on that.  That would probably be better, but can be an activity for a later day.